requirements/doc.txt

```diff
@@ -183,7 +183,7 @@ six==1.16.0
     #   python-dateutil
 snowballstemmer==2.2.0
     # via sphinx
-sphinx==5.2.3
+sphinx==5.3.0
```

Comment:
No breaking change in the new version. For details see Changelog https://www.sphinx-doc.org/en/master/changes.html#release-5-3-0-released-oct-16-2022.